project_key/README.rst

```diff
@@ -0,0 +1,73 @@
+.. image:: https://www.gnu.org/graphics/lgplv3-147x51.png
+   :target: https://www.gnu.org/licenses/lgpl-3.0.en.html
+   :alt: License: LGPL-v3
+
+===========
+Project Key
+===========
+
+This module provides functionality to uniquely identify projects and tasks by simple ``key`` field.
+
+
+Usage
+=====
+
+To use this module functionality you just need to:
+
+On ``project.project`` level:
+
+In Kanban View:
+
+#. Go to Project > Dashboard
+#. Create
+#. Enter project name and use auto generated key or simply override value by entering your own key value.
+
+In Tree View:
+
+#. Go to Project > Configuration > Projects
+#. Create
+#. Enter project name and use auto generated key or simply override value by entering your own key value.
+
+In form View:
+
+#. Go to Project > Dashboard
+#. Open the projects settings
+#. Modify the "key" value
+#. After modifying project key any existing task will be reindexed automatically.
+
+When you create a project, under the hood a ir.sequence record gets creted with prefix: ``<project-key>-``.
+
+On ``project.task`` level:
+
+#. Actually there is nothing to be done here
+#. Task keys are auto generated based on project key value with per project auto incremented number (i.e. PA-1, PA-2, etc)
+
+In browser address bar:
+
+#. Navigate to your project by entering following url: http://<<your-domain>>/browse/PROJECT-KEY
+#. Navigate to your task by entering following url: http://<<your-domain>>/browse/TASK-KEY
```

Comment:
I see you mimicked Jira here that uses `browse` but is this not too specific for Odoo? How about using `tasks` or `tickets`?

Reply:
I've changed it to ``projects`` and ``tasks``